scripts/semconv/generate.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT_DIR="${SCRIPT_DIR}/../../"
+
+# freeze the spec version to make SpanAttributess generation reproducible
+SPEC_VERSION=v1.1.0
+GENERATOR_VERSION=0.2.1
+
+cd ${SCRIPT_DIR}
+
+rm -rf opentelemetry-specification || true
+mkdir opentelemetry-specification
+cd opentelemetry-specification
+
+git init
+git remote add origin https://github.com/open-telemetry/opentelemetry-specification.git
+git fetch origin "$SPEC_VERSION"
```

Comment:
```suggestion
git fetch origin "$SPEC_VERSION" --depth=1
```
as you don't really need the whole history and number of data to download will be the smallest possible

Reply:
I have added this change. Thank you! :)